db/schema.rb

```diff
@@ -37,13 +37,6 @@
     t.datetime "img_updated_at"
   end
 
-  create_table "posts", force: :cascade do |t|
```

Comment:
@bradencanderson  removing the posts table, which we no longer use. Probably not worth asking questions about this file as it's automatically generated.